### PR TITLE
[Improve][API] made optionValues list immutable

### PR DESCRIPTION
**Pull request by anirudh-hegde**
### Purpose of this pull request
made optionValues list immutable to prevent unintended modifications outside the class.

### Does this PR introduce _any_ user-facing change?
No

### CODE REVIEW

**Comment by Carl-Zhou-CN** [comment]
hi, @anirudh-hegde Run 'mvn spotless:apply' to fix these violations.

**Comment by anirudh-hegde** [comment]
> hi, @anirudh-hegde Run 'mvn spotless:apply' to fix these violations.

So I should clone seatunnel project and then run 'mvn spotless:apply' to fix these violations.
Please correct me if I'm wrong.

**Comment by Carl-Zhou-CN** [comment]
> > hi, @anirudh-hegde Run 'mvn spotless:apply' to fix these violations.
> 
> So I should clone seatunnel project and then run 'mvn spotless:apply' to fix these violations. Please correct me if I'm wrong.

To fix these formatting issues, you just need to run the command on your own branch and then commit the code again. It will automatically correct these formatting errors

File: seatunnel-common/src/main/java/org/apache/seatunnel/common/constants/JobMode.java [hunk 1]
```diff
@@ -17,7 +17,9 @@
 
 package org.apache.seatunnel.common.constants;
 
-public enum JobMode {
+import java.io.Serializable;
+
+public enum JobMode implements Serializable {
```
Comment (hailin0) [inline seatunnel-common/src/main/java/org/apache/seatunnel/common/constants/JobMode.java]: revert


image

Comment (anirudh-hegde) [reply seatunnel-common/src/main/java/org/apache/seatunnel/common/constants/JobMode.java]: image
In the above history which to revert

Comment (hailin0) [reply seatunnel-common/src/main/java/org/apache/seatunnel/common/constants/JobMode.java]: revert this file updates: `JobMode.java`

Comment (hailin0) [inline seatunnel-common/src/main/java/org/apache/seatunnel/common/constants/JobMode.java]:
```suggestion
public enum JobMode {
```

**Pull request closed without merging**